seed/utils/match.py

```diff
@@ -429,3 +431,14 @@ def whole_org_match_merge_link(org_id, state_class_name, proposed_columns=[]):
             transaction.set_rollback(True)
 
     return summary
+
+
+def update_sub_progress_total(total, sub_progress_key=None, finish=False):
```

Comment:
Does this belong in /utils/match? It's being used in both /match and /tasks files. It could be in a new /util/progress_data file

Reply:
Could it make sense to put it in `seed/lib/progress_data.py`?

Reply:
I ended up not moving it, this is fine